### ISSUE TITLE
Doesn't work on High Refresh monitors!

### ISSUE THREAD

**Issue by HummusHagen** [ChrisAnd1998/TaskbarX#329]
When I activate the program my monitor goes from 144 to 60, and no, changing the response time on the program does not fix it. I don't think I can use this until it gets fixed. though, this is probably the coolest thing I've done to my computer!

**Comment by ChrisAnd1998** [comment]
Weird, it should adapt to whatever refreshrate you have. Running it on 144hz just fine... I think it might be a bug.

**Comment by HummusHagen** [comment]
Okay, I found that it only has windows at 60, like moving around a window
would display at 60, but the contents within it would be full framerate.
I've restarted my computer a few times since then and haven't noticed the
issue since. I will say though, the blurred taskbar background is not
consistent, Acrylic is, but on blurred, you move a window into it and it's
blurred nicely at the top, but from the middle down it gets less and less
blurred. To the point where I can see sharp lines.

Thank you for responding to this, and thank you for making this taskbar of
eye candy! I will continue using this on my computer, it adds something
different enough it catches my eye.
